e2e_samples/parking_sensors/README.md

```diff
@@ -209,15 +212,37 @@ More resources:
 
 ## How to use the sample
 
-### Prerequisites
+This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.
```

Comment:
please change `development container` to `Dev Container`.

I would simplify this paragraph and just say:
`This sample has certain prerequisites for deployment, which are detailed below. We recommend using a Dev Container to streamline the setup process, ensure consistency across environments, and simplify dependency management.`